web/assets/javascripts/application.js

```diff
@@ -35,7 +31,16 @@ $(function() {
     $($(this).attr('data-target')).toggle();
   });
 
+  $(document).on("click", ".live-poll", function() {
+    localStorage.poll ^= 1;
```

Comment:
Better to make this more descriptive so local storage viewers have some sense of what this variable is for, e.g. `sidekiq_live_poll`?

And maybe this is the Gopher in me but I don't like seeing numeric operations here and boolean operations down below, e.g. `poll = true`. Is this idiomatic?

Reply:
> make this more descriptive

Good call; I added a commit changing this to `localStorage.sidekiqLivePoll`, and the existing `localStorage.timeInterval` to `localStorage.sidekiqTimeInterval` .

Re the numeric/boolean mixing: this was a sloppy way of coping with JS comparisons and the way that localStorage stores values as strings (after setting `localStorage.foo = true;`, `localStorage.foo` returns `"true"` and `"true" != true` but `"1" == true` 🤦). Anyway, I changed it to use "enabled" and "disabled", which is much more clear. Thanks for pointing out the potential for confusion